test/e2e/sgx/sgx.go

```diff
@@ -100,7 +100,7 @@ func describe() {
 					Containers: []v1.Container{
 						{
 							Name:       "testcontainer",
-							Image:      "intel/sgx-sdk-demo:devel",
+							Image:      "intel/sgx-sdk-demo:0.30.0",
```

Comment:
In general demo images are not tied to plugin versions. They are tagged in the release branch because all the images are tagged. "Everyone is treated the same."

Some demo images get more update activities. All of them get updates via their base images.

Reply:
to add to that @tkatila wrote, this aligns with the flow if you `make <demo image>` in the release branch: you get the images tagged with the release version.